framework/src/utils/ImageSampler.C

```diff
@@ -26,8 +26,8 @@ InputParameters validParams<ImageSampler>()
 
   params.addParam<Point>("origin", "Origin of the image (defaults to mesh origin)");
   params.addParam<Point>("dimensions", "x,y,z dimensions of the image (defaults to mesh dimensions)");
-  params.addParam<unsigned int>("component", "The image component to return, leaving this blank will result in a greyscale value "
-                                             "for the image to be created. The component number is zero based, i.e. 0 returns the first component of the image");
+  params.addParam<unsigned int>("component", "The image RGB-component to return, leaving this blank will result in a greyscale value "
```

Comment:
I'll defer to @aeslaughter here. I felt that the doco was a little unclear here as I thought component had something to do with "axis". As you pointed out a MooseEnum may have lots of different options due to different format so you'd have to map the different set of options to overlapping indices. I guess we need to take a look at the VTK API and see what the range of values the API takes. I can drop this if you don't like it.